evaluation.md

```diff
@@ -5,6 +5,51 @@
 _Describe each change from your ideal syntax to the syntax you implemented, and
 describe_ why _you made the change._
 
+To declare a state, we have to pass the string as an argument to the state function. We made this change to keep our implementation time from exploding. 
```

Comment:
I presume you guys thought of having state variables predeclared and rejected that option?